src/util/messageCategory.ml

```diff
@@ -21,12 +21,19 @@ type behavior =
 
 type integer = Overflow | DivByZero [@@deriving eq, ord, hash]
 
+(* Note that this is named '_message' because float itself is a built-in type *)
+type float_message = 
```

Comment:
this type isn't used anywhere. Maybe the `category` should be extended to `FloatMessage of float_message` ?